src/main/java/com/marcusslover/plus/lib/text/Text.java

```diff
@@ -92,6 +93,16 @@ private Text(@NotNull Component component) {
         return this;
     }
 
+    /**
+     * Deserializes text using MiniMessage.
+     * @return MiniMessage deserialized component.
+     */
+    public @NotNull Text mini() {
+        MiniMessage mm = MiniMessage.miniMessage();
```

Comment:
Hey thanks a cool addition, can we make it so instead make like a static function Text.mini("etc");

Reply:
Forgot to make it static, done